{
    "CVE_data_meta": {
        "ASSIGNER": "security-advisories@github.com",
        "ID": "CVE-2022-36046",
        "STATE": "PUBLIC",
        "TITLE": "Unexpected server crash in Next.js version 12.2.3"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "next.js",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "= 12.2.3"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "vercel"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "Next.js is a React framework that can provide building blocks to create web applications. All of the following must be true to be affected by this CVE: Next.js version 12.2.3, Node.js version above v15.0.0 being used with strict `unhandledRejection` exiting AND using next start or a [custom server](https://nextjs.org/docs/advanced-features/custom-server). Deployments on Vercel ([vercel.com](https://vercel.com/)) are not affected along with similar environments where `next-server` isn't being shared across requests."
            }
        ]
    },
    "impact": {
        "cvss": {
            "attackComplexity": "HIGH",
            "attackVector": "NETWORK",
            "availabilityImpact": "HIGH",
            "baseScore": 5.3,
            "baseSeverity": "MEDIUM",
            "confidentialityImpact": "NONE",
            "integrityImpact": "NONE",
            "privilegesRequired": "LOW",
            "scope": "UNCHANGED",
            "userInteraction": "NONE",
            "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:N/A:H",
            "version": "3.1"
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-248: Uncaught Exception"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "name": "https://github.com/vercel/next.js/security/advisories/GHSA-wff4-fpwg-qqv3",
                "refsource": "CONFIRM",
                "url": "https://github.com/vercel/next.js/security/advisories/GHSA-wff4-fpwg-qqv3"
            },
            {
                "name": "https://github.com/vercel/next.js/releases/tag/v12.2.4",
                "refsource": "MISC",
                "url": "https://github.com/vercel/next.js/releases/tag/v12.2.4"
            }
        ]
    },
    "source": {
        "advisory": "GHSA-wff4-fpwg-qqv3",
        "discovery": "UNKNOWN"
    }
}